about.html

```diff
@@ -0,0 +1,130 @@
+                        <li><a href="https://www.linkedin.com/in/gerson-sanchez-88309b57/"><img
+                                    src="img\Linkedin icon.svg" alt="logo linkedin"></a></li>
+                        <li><a href="https://angel.co/"><img src="img\angel list.svg" alt="logo angel list"></a></li>
+                        <li><a href="https://twitter.com/gealsanchez"><img src="img\twitter.svg" alt="logo twitter"></a></li>
+                        <li> <a href="https://medium.com/tag/javascript"><img src="img\medium.svg" alt="logo medium"></a></li>
+                    </ul>
+
+                </div>
+
+                <div class="desktopmenu d-flex justify-content-between">
+                    <a class="ms-5 ps-5 logo" href="index.html"><img src="img/logo.png" alt="logo creative commons"></a>
+                    <ul>
+                        <a style="color: rgb(255, 51, 0);" href="index.html">
+                            <li>Home</li>
+                        </a>
+                        <a href="#">
+                            <li class="ms-5">Program</li>
+                        </a>
+                        <a href="#">
+                            <li class="ms-5">Join</li>
+                        </a>
+                        <a href="#">
+                            <li class="ms-5">Sponsor</li>
+                        </a>
+                        <a href="#">
+                            <li class="ms-5 me-3">News</li>
+                        </a>
+                        <a style="border: 3px solid rgb(255, 51, 0); border-radius: 5px;color: rgb(255, 51, 0);"
+                            href="#">
+                            <li class="mx-5">Campaign</li>
+                        </a>
+                    </ul>
+                </div>
+
+            </nav>
+        </header>
+
+        <div class="container text-dark my-2 pb-1">
+            <h3 class="pt-5 pb-2 mt-2" style="color: rgb(255, 51, 0); font-weight: 400;font-size: 30px;">"Hello, Sharing
+                world"</h3>
+            <h1 style="color: rgb(255, 51, 0);">CONTAINER FLUID</h1>
+            <h1 style="color: rgb(255, 51, 0);">ARTS FESTIVAL</h1>
+            <h1 style="color: rgb(255, 51, 0);">2022</h1>
+            <p class="bg-light my-3 p-1" style="font-size: 18px;width: 100%;">The mission of the Container Festival is to
+                promote the arts and enhance the
+                quality of life through the production of an annual outdoor, multi-disciplinary event. We strive to
+                maintain the highest artistic quality in our production and programming, represent excellence among a
+                range of artistic mediums, and promote an appreciation for fine art while encouraging innovation in the
+                field, and supporting non-traditional, contemporary works</p>
+        </div>
+        <a class="logo text-center" style="margin-bottom:5%;" href=""><img src="img/logo.png" alt="logo creative commons"></a>
```

Comment:
- [ ] For the logo section in the about page, kindly try to follow the design guidelines as much as possible. There should be a header, a text and the logo 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/198694390-bba6963a-1a9e-4be2-b06e-913fadcd35e2.png)

## Design guidelines:
![image](https://user-images.githubusercontent.com/50721479/198694430-9f1eb2c6-f714-49d1-95fc-15493b5874ff.png)